ingestion_server/ingestion_server/elasticsearch_models.py

```diff
@@ -280,6 +269,19 @@ def get_aspect_ratio(height, width):
             aspect_ratio = Image.AspectRatios.SQUARE.name
         return aspect_ratio.lower()
 
+    @staticmethod
+    def get_extension(url):
```

Comment:
```suggestion
    def get_extension(filetype, url):
```

Reply:
We would also need to change line 231 above to `extension = Image.get_extension(row[schema["filetype"]], row[schema["url"]])` (adding the filetype as a parameter).

Reply:
Why these changes? I wanted to keep image extension changes minimal on this PR.